docs/pages/debugging/create-devtools-plugins.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Create a dev tools plugin
+description: Learn how to create a dev tools plugin to enhance your development experience.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { APIBox } from '~/components/plugins/APIBox';
+
+> **warning** Dev Tools plugins are compatible with SDK 50 and above.
+
+You can create a dev tools plugin, whether that's for inspecting aspects of a common framework or library or something specific to your custom code. This guide will walk you through creating a dev tools plugin.
+
+## What is a dev tools plugin?
+A dev tools plugin runs in your web browser in your local development environment and connects to your Expo app.
+
+A plugin consists of three key elements:
+- An Expo app to display the dev tools web user interface.
+- An **expo-module.config.json** for Expo CLI recognition.
+- Calls to `expo/devtools` API for the app to communicate back and forth with the dev tool's web interface.
+
+Plugins can be distributed on npm or included inside your app's monorepo. They typically export a single hook that can be used in your app's root component to initiate two-way communication with the web interface when your app is running in debug mode.
+
+## Create a plugin
+
+### Create a new plugin project
+
+`create-dev-plugin` will setup a new plugin project for you. Run the following command to create a new plugin project:
+
+<Terminal cmd={['$ npx create-dev-plugin']} />
```

Comment:
```suggestion
<Terminal cmd={['$ npx create-dev-plugin@latest']} />
```

recommend to use the latest dist-tag for npx, in case npx cache the package and keep using the old version